recipes/nltk-data/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+NTLK_DATA=$PREFIX/nltk_data
+mkdir -vp $NTLK_DATA
+
+# Download from the source of the package, its on the $SRC_DIR
+
+mv $SRC_DIR/packages/* $NTLK_DATA
+find $NTLK_DATA/ -name "*.zip" | while read filename; do unzip -qq -o -d "`dirname "$filename"`" "$filename"; done;
```

Comment:
`find $NLTK_DATA/ -name "*.zip" -execdir unzip -qq -o '{}' ';' -delete` would be a little faster, and would also delete the zip files afterwards (saving the second `find` below).

Reply:
Done. Thanks for the better command. took me a minute to get one that worked.